Adapptr/Sample-Code&Scripts/Python/adapptr.py

```diff
@@ -0,0 +1,94 @@
+from typing import Any, Dict, List, Optional, Tuple
+import requests
+import sys
+from datetime import datetime
+import os
+import time
+
+tenant_name = "insert your tenant name"
+base_url = f"https://{tenant_name}-svc.fundapps.co/api/adapptr/v2/"
+
+username = os.environ.get("ADAPPTR_USERNAME")
+password = os.environ.get("ADAPPTR_PASSWORD")
+
+def upload_file(file_path: str, snapshote_date: datetime.date) -> str:
+    """
+    Posts a position file for a specific date.
+    
+    Args:
+        file_path (str): The path to the file to be posted.
+        snapshot_date (datetime.date): The date that the position data relates to
+        
+    Returns:
+        Task Id: The Adapptr id for the run (used to check status)
+    """
+    url = f"{base_url}task/positions"
+    data = {
+        'snapshotDate': snapshote_date.strftime('%Y-%m-%d'),
+        'dataProvider': 1, # 1 for Refinitiv, 2 for Bloomberg
```

Comment:
Could we link to the version docs, instead of stating them here? Doubt the values are gonna change, but, golden source